administrator/language/de-DE/plg_fields_menuitem.ini

```
@@ -0,0 +1,9 @@
+; Joomla! German Translation
+; (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2022 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_FIELDS_MENUITEM="Felder - Menüeintrag"
```

```suggestion
PLG_FIELDS_MENUITEM="Felder – Menüeintrag"
```
Typografisch sollte hier ein enDash stehen, statt eines Divis

---

@max123kl 
OK. Dann solltest du ggf. bei allen Plugins das mal als PR einreichen.

---

Hieße das, dass die komplette Übersetzung geprüft werden sollte?

---

> Hieße das, dass die komplette Übersetzung geprüft werden sollte?

Macht sicherlich Sinn, da ich es nirgends finden konnte.
Kann man ja grad bei den Plugins mit Search&Replace in einer IDE machen.

---

OK